tests/typegen/strats/test_inline.py

```diff
@@ -700,7 +700,7 @@ def function(parameter: Clazz):
     def __init__(self):
         self.class_member: int = 5
 
-    def change_value(self, parameter1: int, parameter2: str) -> None:
+    def change_value(self, parameter1: int, parameter2: str) -> bool:
```

Comment:
It seems like you have found a bug, so I added bug issue #159. You do not need to review this comment.